plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -84,7 +84,7 @@ def ray_rsync_down(yaml_path: str, remote_dir: str, local_dir: str) -> None:
 def ray_tmp_dir(yaml_path: str, docker: bool) -> Generator[Any, None, None]:
     args = ["ray", "exec"]
     if docker:
-        args.append("--docker")
+        args.append("--run-env=docker")
```

Comment:
what is the difference?
Is this the deprecation thing you were reporting?

Reply:
correct. As of Ray v0.8.7 `--docker` is not the correct flag for `ray exec`